recipes/rich-click/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "rich-click" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ewels/rich-click/archive/v{{ version }}.tar.gz
+  sha256: 6204e3ae465abca7cf630649d9c9b7853d4ec404c85eeab03b8af77db91616f3
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - rich >=10
+    - click >=7.0
```

Comment:
I don't think you need the `.0` here. So, `>=7,<8` maybe?

Reply:
Ah I went back and forth on the `.0`! I can certainly remove that 👍🏻 

What's the reason for the `<8` though? It works fine with v8, just needs at least v7 or later..

Reply:
Sorry, `<9`, i.e., the next unreleased version.

Reply:
```suggestion
    - click >=7.0,<9
```

Reply:
```suggestion
    - click >=7,<9
```